### PR TITLE
Header: add Incubator link

### PR DESCRIPTION
**Pull request by breville**
This change adds a link to the Incubator (https://studio.code.org/incubator) to our header and hamburger across the site, for all "en" visitors. 

It shows for signed-out users, as well as signed-in students and teachers.  It's shown in the header along with other options, or in the hamburger on narrower displays.  To prevent overlaps, the "Create" button in the header is now shown for browser widths of 1101px of wider, up from 1024px before.

Signed-in teacher on pegasus, full width:
Screenshot 2023-02-01 at 6 15 20 PM

Signed-in teacher on pegasus, medium width:
Screenshot 2023-02-01 at 6 16 33 PM

Signed in-teacher on pegasus, narrow width:
Screenshot 2023-02-01 at 6 17 35 PM

Signed-in teacher on dashboard, full width:
Screenshot 2023-02-01 at 6 18 39 PM

Signed-in teacher on dashboard, medium width:
Screenshot 2023-02-01 at 6 19 09 PM

Signed-in teacher on dashboard, narrow width:
Screenshot 2023-02-01 at 6 20 19 PM

Signed-out, full width:
Screenshot 2023-02-01 at 6 24 02 PM

Signed-out, narrow width:
Screenshot 2023-02-01 at 6 25 17 PM

Signed-in student, full width:
Screenshot 2023-02-01 at 6 30 47 PM

Signed-in student, narrow width:
Screenshot 2023-02-01 at 6 31 02 PM

### CODE REVIEW

File: lib/cdo/hamburger.rb [hunk 1]
```diff
@@ -113,12 +113,27 @@ def self.get_hamburger_contents(options)
...
         teacher_entries << {
           title: I18n.t("#{loc_prefix}professional_learning"),
           url: CDO.studio_url("/my-professional-learning"),
         }
+
+        teacher_entries << {
+          title: I18n.t("#{loc_prefix}incubator"),
+          url: CDO.studio_url("/incubator"),
+        }
+      elsif options[:user_type] == "student"
```
Comment (sanchitmalhotra126) [inline lib/cdo/hamburger.rb]: nit: Do these need to be conditionally added? Could we just iterate over all entries and add the incubator entry, similar to how international is added below?

**Review by sanchitmalhotra126 — APPROVED** [review]
One question but LGTM otherwise!

File: lib/cdo/hamburger.rb (hunk 1 above)
Comment (wilkie) [reply lib/cdo/hamburger.rb]: They are all the same entry, right? I agree with doing something similar to how international is added.

**Review by wilkie — APPROVED** [review]
I agree that the change Sanchit recommends feels right to tighten up the logic and maintain the link in just one place. Though beyond that... looks good! 👍

File: lib/cdo/hamburger.rb (hunk 1 above)
Comment (breville) [reply lib/cdo/hamburger.rb]: Good call!

**Pull request merged**